composer.json

```diff
@@ -27,28 +27,33 @@
     "config": {
         "sort-packages": true
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/webimpress/zend-expressive-router.git"
+        }
+    ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
         "fig/http-message-util": "^1.1.2",
-        "http-interop/http-middleware": "^0.4.1",
+        "http-interop/http-server-middleware": "^1.0.1",
         "psr/container": "^1.0",
         "psr/http-message": "^1.0.1",
-        "webimpress/http-middleware-compatibility": "^0.1.1",
         "zendframework/zend-diactoros": "^1.3.10",
-        "zendframework/zend-expressive-router": "^2.2",
+        "zendframework/zend-expressive-router": "dev-feature/psr-15 as 2.2",
```

Comment:
You can now pin this to 3.0.x-dev or dev-release-3.0.0.

Reply:
Unfortunately for now we need to have `3.0.x-dev as 2.3` as expressive-*router libraries does not support v3 yet. I'm going to work on it soon.